components/sst_importer/src/sst_importer.rs

```diff
@@ -664,21 +664,14 @@ impl ImportDir {
         let cf = meta.get_cf_name();
         super::prepare_sst_for_ingestion(&path.save, &path.clone, key_manager)?;
         let length = meta.get_length();
-        let crc32 = meta.get_crc32();
-        // FIXME perform validate_sst_for_ingestion after we can handle sst file size correctly.
-        // currently we can not handle sst file size after rewrite,
-        // we need re-compute length & crc32 and fill back to sstMeta.
-        if length != 0 {
-            if crc32 != 0 {
-                // we only validate if the length and CRC32 are explicitly provided.
-                engine.validate_sst_for_ingestion(cf, &path.clone, length, crc32)?;
-            }
-            IMPORTER_INGEST_BYTES.observe(length as _)
-        }
+        // length & crc32 is no longer used in validate_sst_for_ingestion any more.
+        engine.validate_sst_for_ingestion(cf, &path.clone, 0, 0)?;
```

Comment:
perhaps we should just apply the TODO and rename `validate_sst_for_ingestion` to `reset_global_seq` to avoid the confusion why you pass 0 to the length & crc32 fields.

https://github.com/tikv/tikv/blob/79a0cfbca2c9015e9fe804505ce91d6c70cf04c4/components/engine_rocks/src/import.rs#L34-L35

Reply:
Done, PTAL